app/controllers/settings_controller.rb

```diff
@@ -1,7 +1,7 @@
 class SettingsController < ApplicationController
-  include SessionsHelper
+  include SettingsHelper
 
-  before_action :authentication_required
+  #before_action :authentication_required
```

Comment:
There appears to be no path to the settings tab unless you are logged in...

On Thu, Apr 9, 2015 at 1:14 PM, Ben Woodall <notifications@github.com>
wrote:

> In app/controllers/settings_controller.rb
> <https://github.com/benwoody/shrtnr/pull/4#discussion_r28098175>:
>
> >
> > -  before_action :authentication_required
> > +  #before_action :authentication_required
>
> So what happens if I'm NOT signed in and I visit /settings ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benwoody/shrtnr/pull/4/files#r28098175>.
>



-- 
Joe Simpson
“Reasonable people adapt themselves to the world. Unreasonable people
attempt to adapt the world to themselves. All progress, therefore, depends
on unreasonable people.”
George Bernard Shaw